types/svg.d.ts

```diff
@@ -1,4 +1,4 @@
 declare module '*.svg' {
-    const content: React.SVGAttributes<SVGElement>;
+    const content: React.FC<React.SVGAttributes<SVGElement>>;
```

Comment:
```suggestion
    const content: React.ComponentType<React.SVGAttributes<SVGElement>>;
```

Reply:
@kate-deriv I think you need to change in one of you PRs. So it doesn't get overwritten.